src/utils/pathwayAnalysis/usePantherDBSpecies.js

```diff
@@ -0,0 +1,31 @@
+import useSWR from 'swr';
+
+const formatOuput = (data) => {
+  const { genome } = data.search.output.genomes;
+
+  const speciesList = genome.map((species) => ({
+    label: species.long_name,
+    value: species.short_name,
+  }));
+
+  return speciesList;
+};
+
+const usePantherDBSpecies = () => {
+  const { data, error } = useSWR(
+    'http://pantherdb.org/services/oai/pantherdb/supportedgenomes',
+    async (url) => {
+      const response = await fetch(url);
+      const responseData = await response.json();
+      return responseData;
+    },
+  );
+
+  const formattedData = data ? formatOuput(data) : [];
+  return {
+    data: formattedData,
+    error,
+  };
+};
+
+export default usePantherDBSpecies;
```

Comment:
Nice! Although for a single use case (we don't expect this to be used anywhere else, right?) I would probably say it didn't need the extra work to extract it into a special hook, perhaps just wrap the fetch and parsing into a function for readibility and that's it.